engine/engine_test.go

```diff
@@ -71,3 +71,19 @@ func TestLoadConfigWithSettings(t *testing.T) {
 		})
 	}
 }
+func TestStartStopDoesNotCausePanic(t *testing.T) {
```

Comment:
Could you please add one new line above here?

Reply:
Done. Should add a rule to linter?

Reply:
This is covered by gofumpt which can be enabled on golangci-lint, but out of scope for this PR